sql/plan/show_create_table.go

```diff
@@ -453,3 +404,77 @@ func produceCreateViewStatement(view *SubqueryAlias) string {
 func (i *showCreateTablesIter) Close(*sql.Context) error {
 	return nil
 }
+
+func CreateTableFmt(tblName string, colStmts []string, tblCharsetName, tblCollName string) string {
```

Comment:
This utility functions would be a better fit in the `sql` package (or even in a new `sqlfmt` package under `sql` to match the `sqlfmt` package in Dolt that has similar code, although that's really overkill right now). `plan` is a little odd as a package for these, because they aren't related to query planning – they are more of utility functions for generating sql statements. I would try moving these to `sql`, perhaps in a `sqlfmt.go` file, and see how they look. 

It would also be good to think about how to make these functions feel connected and cohesive. The "Fmt" prefix is a good start, but it's a little inconsistent in the naming, and I think something like "Generate" might read better (even though it's more verbose). Something like these could be nice: "GenerateCreateTableStatement", "GenerateCreateTableColumnDefinition", "GenerateCreateTableConstraintClause", etc. 

Giving these functions some doc strings would also help people more easily understand how to use them.